rpcs3/Emu/Io/pad_types.h

```diff
@@ -134,6 +134,9 @@ enum
 	CELL_PAD_PCLASS_TYPE_DANCEMAT   = 0x04,
 	CELL_PAD_PCLASS_TYPE_NAVIGATION = 0x05,
 	CELL_PAD_PCLASS_TYPE_SKATEBOARD = 0x8001,
+	CELL_PAD_PCLASS_TYPE_GUNCON3    = 0x8002,
```

Comment:
This makes no sense unless you can prove that it's reported as 0x8002 on real hardware

Reply:
I changed its name and a picked a higher value to avoid suggesting it's a real value.

Reply:
What's your suggestion?